akka-projection-kafka/src/main/scala/akka/projection/kafka/internal/KafkaSourceProviderImpl.scala

```diff
@@ -67,6 +67,7 @@ import org.apache.kafka.common.record.TimestampType
   private[kafka] val partitionHandler = new ProjectionPartitionHandler
   private val scheduler = system.classicSystem.scheduler
   private val subscription = Subscriptions.topics(topics).withPartitionAssignmentHandler(partitionHandler)
+  private[projection] var control: Option[Consumer.Control] = None
```

Comment:
In preparation of https://github.com/akka/akka-projection/issues/342

I'll just leave this impl. intact for now, even if it's not called.